Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionSonicWallFirewall.yaml

```diff
@@ -0,0 +1,413 @@
+    , sosAppSignatureID = sid // Application Signature ID
+    , sosIPSCategoryName = ipscat // IPS Category Name
+    , sosAntiSpywareCategory = spycat // Anti-Spyware Category
+    , sosURLPathName = arg // URL. Represents the URL path name.
+    , sosFileIdentifier = fileid // File hash or URL
+    , sosDPIInspectedFlow = dpi // Indicates a flow was inspected by DPI. Applies only to Connection Closed messages.
+    , DstNatPortNumber = dnpt
+    , SrcNatPortNumber = snpt
+    , sosBladeID = bid // Blade ID
+    , sosUUID = uuid
+    , sosHttpRequestMethod = HttpRequestMethod
+    , sosFileName = FileName
+    , DvcOriginalAction = fw_action
+  | extend
+    ThreatName = coalesce(sosASWSignatureName, sosGAVSignatureName, sosIPSSignatureName, "")
+    , ThreatId = coalesce(sosAppSignatureID, "")
+    , ThreatCategory = coalesce(sosIPSCategoryName, sosAntiSpywareCategory, "")
+    , DstNatPortNumber = toint(DstNatPortNumber)
+    , SrcNatPortNumber = toint(SrcNatPortNumber)
+  | extend AdditionalFields = bag_pack(
+    "ReceivedPackets", sosReceivedPackets
+    , "SentPackets", sosSentPackets
+    , "ConnectionDuration", sosConnectionDuration
+    , "AppRulePolicyId", sosAppRulePolicyId
+    , "AppRulePolicyName", sosAppRulePolicyName
+    , "AppRuleService", sosAppRuleService
+    , "AppRuleType", sosAppRuleType
+    , "AppRuleObject", sosAppRuleObject
+    , "AppRuleObjectContent", sosAppRuleObjectContent
+    , "AppRuleAction", sosAppRuleAction
+    , "AppIDNumber", sosAppIDNumber
+    , "AppID", sosAppID
+    , "AppCategoryID", sosAppCategoryID
+    , "AppSignatureID", sosAppSignatureID
+    , "IPSCategoryName", sosIPSCategoryName
+    , "AntiSpywareCategory", sosAntiSpywareCategory
+    , "URLPathName", sosURLPathName
+    , "FileIdentifier", sosFileIdentifier
+    , "DPIInspectedFlow", sosDPIInspectedFlow
+    , "BladeID", sosBladeID
+    , "UUID", sosUUID
+    , "HttpRequestMethod", sosHttpRequestMethod
+    , "FileName", sosFileName
+    , "FileSize", FileSize
+    , "HttpReferrer", sosHttpReferrer
+    , "HttpUserAgent", HttpUserAgent
+    , "CaptureATPVerdict", sosCaptureATPVerdict
+    , "ApplicationID", sosApplicationID
+    , "CFSCategoryID", sosCFSCategoryID
+    , "CFSCategoryName", sosCFSCategoryName
+    , "CFSPolicyName", sosCFSPolicyName
+    , "AppControlFileName", sosAppControlFileName
+    , "AppCategoryName", sosAppCategoryName
+    , "AppIDName", sosAppIDName
+    , "AppSignatureName", sosAppSignatureName
+    , "IPSFullString", sosIPSFullString
+    , "IPSSignatureName", sosIPSSignatureName
+    , "LegacyMessageCategory", sosLegacyMessageCategory
+    , "LogMsgCategory", sosLogMsgCategory
+    , "LogMsgNote", sosLogMsgNote
+    , "LogMsgSeverity", sosLogMsgSeverity
+    , "SourceVPNPolicyName", sosSourceVPNPolicyName
+    , "DestinationVPNPolicyName", sosDestinationVPNPolicyName
+    , "SerialNumber", sosSerialNumber
+    , "DeviceModel", sosDeviceModel
+    , "LogMsgID", sosLogMsgID
+    , "RequestURL", RequestURL

```

Comment:
this is a WebSession field. I don't think you'll find this populated in an NetworkSession log.